index.html

```diff
@@ -95,26 +96,43 @@ <h1>
                                 <li><a href=#></a></li>
                                 
                             </ul>
-                        </div>
+                        </div> -->
                         
 
                     </div>
-                    <div class="col-lg-9 main_two" style="background-size: 100%; background-image:url('images/bag.jpg');">
+                    <div class="col-12 main_two text-center d-flex align-items-center justify-content-around" style="background-size: 100%; max-width: 100%; height: 500px;  background-image:url('images/bag.jpg');">
                         <div class="main__item" >
                             <div class="main__text">
-                                <span>FRUIT FRESH</span>
-                                <h2>Vegetable <br />100% Organic</h2>
-                                <p>Free Pickup and Delivery Available</p>
-                                <a href="#" class="primary-btn">SHOP NOW</a>
+                                <h2>Find <br />all discounts</h2>
+                                <p>Only make smart decisions</p>
+                                <a href="#" class="btn btn-sample-one">SAVE YOUR MONEY</a>
                             </div>
                         </div>
                     </div>
 
+                    <div id="content">
+                        <div class="card" style="width: 18rem;">
+                            <img class="card-img-top" src="images/bag.jpg" alt="Card image cap">
+                            <div class="card-body">
+                              <h5 class="card-title">Heidelbeeren</h5>
+                              <p class="card-text">Description</p>
+                            </div>
+                            <ul class="list-group list-group-flush">
+                              <li class="list-group-item font-weight-bold">Price: <span class="text-danger h4">48 Euro</span></spand></li>
+                              <li class="list-group-item text-muted">Basic Price: <span class="text-danger h4">48 Euro</span></spand></li>
+                            </ul>
+                            <div class="card-body">
+                              <a href="#" class="card-link">Add to Basket</a>
+                            </div>
+                        </div>
+                    </div>
                 </div>
             </div>
         </section>
 
-        <div class="row">
+
+
+        <!-- <div class="row">
```

Comment:
Same here and please check the rest of the code for not used and commented code.